DatadogCore/Sources/Core/DatadogCore.swift

```diff
@@ -529,6 +531,8 @@ extension DatadogContextProvider {
         subscribe(\.brightnessLevel, to: BrightnessLevelPublisher(notificationCenter: notificationCenter))
         #endif
 
+        subscribe(\.localeInfo, to: LocaleInfoPublisher(initialLocale: LocaleInfo(), notificationCenter: notificationCenter))
```

Comment:
We rather want to keep it in-sync with what was passed to `DatadogContext`, right?
```suggestion
        subscribe(\.localeInfo, to: LocaleInfoPublisher(initialLocale: locale, notificationCenter: notificationCenter))
```